internal/hud/prompt/prompt.go

```diff
@@ -203,8 +209,16 @@ func (p *TerminalPrompt) OnChange(ctx context.Context, st store.RStore) {
 					msg.stopCh <- true
 
 				case 'h':
+					warningMsg := "We're removing the HUD key shortcut from the prompt.\n" +
+						"In future releases, open the HUD with: tilt up --hud=true"
+					_, _ = fmt.Fprintf(p.stdout, "%s\n", warningMsg)
+
+					time.Sleep(p.hudDelay)
```

Comment:
A 6-second delay seems like it'd be really annoying! Would it be cheap to do something like "press h again to launch the HUD"?